src/cf-forms/src/cf-forms.less

```diff
@@ -219,6 +219,663 @@ textarea {
+          </div>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            .form-l
+              .form-l_col.form-l_col-1-3
+        notes:
+          - ".form-l_col-1-3 elements are stacked for small screens. When the
+            viewport reaches 600px they transform into columns with one half the
+            width of the container. When the viewport reaches 768px their width
+            updates to one third of the container."
+      - name: "Form layout modifier: flush"
+        markup: |
+          <div class="form-l form-l__flush">
+              <div class="form-l_col form-l_col-1-3">
+                  Form layout column 1
+              </div>
+              <div class="form-l_col form-l_col-1-3">
+                  Form layout column 2
+              </div>
+              <div class="form-l_col form-l_col-1-3">
+                  Form layout column 3
+              </div>
+          </div>
+        codenotes:
+          - .form-l.form-l__flush
+        notes:
+          - "Form layout columns have left and right gutters. If you want the
+            first and last column in each row to sit flush left/right
+            respectively then use the .form-l__flush modifier."
+      - name: Form layout inset
+        markup: |
+          <div class="form-l form-l__flush">
+              <div class="form-l_col form-l_col-1-2 form-l-inset">
+                  <div class="form-l-inset_container">
+                      <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default-inset" id="cf-input-example_default-inset">
+                      <label class="cf-input_label" for="cf-input-example_default-inset">
+                          <span class="cf-input_text">Yes</span>
+                      </label>
+                      <span class="cf-input_radio"></span>
+                  </div>
+              </div>
+              <div class="form-l_col form-l_col-1-2 form-l-inset">
+                  <div class="form-l-inset_container">
+                      <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default-inset" id="cf-input-example_default-inset2">
+                      <label class="cf-input_label" for="cf-input-example_default-inset2">
+                          <span class="cf-input_text">No</span>
+                      </label>
+                      <span class="cf-input_radio"></span>
+                  </div>
+              </div>
+          </div>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            .form-l-inset
+              .form-l-inset_container
+        notes:
+          - ".form-l-inset_container elements have larger target areas for easier user interaction."
+    tags:
+      - cf-forms
+*/
+
+.form-l-inset {
+    border-right-width: 2.5px;
+    + .form-l_col-1-2.form-l-inset:nth-child(odd) { border-left-width: 2.5px; }
```

Comment:
Add line breaks between rules for better readability and put rule contents on new lines for consistency.